processor/src/main/java/org/mapstruct/ap/internal/model/PropertyMapping.java

```diff
@@ -180,6 +189,15 @@ public T existingVariableNames(Set<String> existingVariableNames) {
             this.existingVariableNames = existingVariableNames;
             return (T) this;
         }
+
+
+        protected static boolean isConstructorMapping(SourceMethod method, String targetPropertyName) {
```

Comment:
I need to remove this method, right now it is required because I don't have access to the TargerWriteAccessorType object because it is not saved on the property mapping. I could add it but I am not sure if this is the best approach